Intact.BuinessLogic/Models/Spell.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Text.Json.Serialization;
+using Intact.BusinessLogic.Converters;
+using Intact.BusinessLogic.Data.Enums;
+
+namespace Intact.BusinessLogic.Models;
+
+public record Spell: Localizable
+{
+    [JsonPropertyName("spell")]
+    public string Mechanics  { get; set; }
+    
+    [JsonConverter(typeof(JsonPrefixEnumConverter<SpellTargetType>))]
+    public SpellTargetType Target { get; set; }
+
+    [JsonConverter(typeof(JsonPrefixEnumConverter<SpellUseType>))]
+    public SpellUseType Use { get; set; }
+
+    [JsonConverter(typeof(JsonPrefixEnumConverter<SpellKind>))]
+    public SpellKind Kind { get; set; }
```

Comment:
SpellKindType would be more correct in my opinion.

Reply:
Firstly,  I have wanted SpellKindType, but i was confused because "Kind" and "Type" are similar, why not SpellKindKind, or SpellTypeType? "SpellType" is wrong, because the field name "Kind". In essential, it's not type OF kind, it's type OF spell.

Reply:
I removed redundant postfixes, it makes more sense to me without them

Reply:
As I suggested in the chat, we could use a naming idea that I had some years before, for this spellKind to name it spellNature (physical or magical)